File: python_ToSXs/python/07-simdienthoai/91.py
phone = "0917.934.678"

result = "No"
phone = phone.replace(".", "")
length = len(phone)
phone = phone[length - 3:]

first = int(phone[0])
second = int(phone[1])
third = int(phone[2])

if first == second - 1 and first == third - 2:
    result = "Yes"

print(result)

File: cpp_e9zMZ/cpp/07-simdienthoai/91.cpp
#include <iostream>

#include "zvn.h"
using namespace std;

int main() {
    string phone = "0589244567";
    string result = "No";

    phone = myReplace(phone, ".");
    phone = phone.substr(phone.length() - 3);

    int first = phone[0] - 0;
    int second = phone[1] - 0;
    int third = phone[2] - 0;

    if (first == second - 1 && first == third - 2) result = "Yes";

    cout << result << endl;
    return 0;
}